code/code/spec/spec_objs_weapons.cc

```diff
@@ -2748,3 +2748,196 @@ int glacialWeapon(TBeing *vict, cmdTypeT cmd, const char *arg, TObj *o, TObj *)
+       0, ch, o, 0, TO_ROOM);
+
+       act("<w>Your $o crackles with <b>blue electricity<z> arching into the $g.<1>",
+       0, ch, o, 0, TO_CHAR);
+
+     }
+ }
+
+// Combat procs
+  if(cmd == CMD_OBJ_HIT)
+    {
+// Slightly lowering chance to proc to balance with bone break/unmaker
+// Since this has extra damage components
+      chance = ::number(1,57);
+      if (!o || !vict)
+      return FALSE;
+
+// Witherlimb proc + 10-20 damage if successful (max once/fight)
+
+      if (chance == 1) {
+      if(!ch->canWither(vict, SILENT_YES))
+      return false;
+
+      bool ok = false;
+	
+      bool found = false;
+  
+// Initial checks to prevent issues and for balancee reasons
+      for (slot = MIN_WEAR; slot < MAX_WEAR; slot++) {
+      if (notBreakSlot(slot, false)) // same ones, right?
+      continue;
+
+      if (!vict->slotChance(slot))
+      continue;
+
+      found |= (vict->isLimbFlags(slot, PART_MISSING));
+
+      ok = true;
+    }
+
+  if (found || !ok) 
+  return false;
+
+// Accounting for electricity immunity since this is electricity-based wither limb
+  if (vict->isImmune(IMMUNE_ELECTRIC, slot))
+  return false;
+
+// Get a random limb slot and make sure it's valid
+  for (slot = pickRandomLimb();; slot = pickRandomLimb()) {
+    if (notBreakSlot(slot, true))
+    continue;
+
+    if (!vict->slotChance(slot))
+    continue;
+
+    break;
+
+    }
+
+  if (!vict->hasPart(slot)) {
+    vlogf(LOG_COMBAT, format("BOGUS SLOT trying to be made PART_USELESS: %d on %s") % 
+    slot % vict->getName());
+
+    return FALSE;
+
+    }
+
+  if (!vict->roomp) {
+    vlogf(LOG_COMBAT, format("!roomp for target (%s) when trying to make PART_USELESS.") % vict->getName());
+    return FALSE;
+
+    }
+
+  vict->setLimbFlags(slot, PART_MISSING);
+
+  TThing *t;
+
+  if ((t = vict->unequip(slot)))
+  *(vict->roomp) += *t;
+
+  for (wearSlotT j=MIN_WEAR; j < MAX_WEAR; j++) {
+    if (!vict->hasPart(j))
+    continue;
+
+    if (!vict->limbConnections(j)) {
+      vict->setLimbFlags(j, PART_MISSING);
+      TThing *tmp = vict->unequip(j);
+
+      if (tmp)
+      *(vict->roomp) += *tmp;
+
+    }
+  }
+// check for damage to both hands
+
+  vict->woundedHand(TRUE);
+
+  vict->woundedHand(FALSE);
+
+  sprintf(limb, "%s", vict->describeBodySlot(slot).c_str());\
+  buf = format("Your $p flashes with a blinding white light and emits a bolt of lightning!");
+
+  act(buf, FALSE, ch, o, vict, TO_CHAR, ANSI_WHITE);
+  buf = format("$n's $p flashes with a blinding white light and emits a bolt of lightning!");
+
+  act(buf, FALSE, ch, o, vict, TO_NOTVICT, ANSI_WHITE);
+  buf = format("$n's $p flashes with a blinding white light and emits a bolt of lightning!");
+
+  act(buf, FALSE, ch, o, vict, TO_VICT, ANSI_WHITE);
+
+  buf = format("You look down in horror and disbelief, as $n's $p turns your %s into ash!") % limb;
+
+  act(buf, FALSE, ch, o, vict, TO_VICT, ANSI_GREY);
```

Comment:
https://github.com/sneezymud/sneezymud/blob/master/code/code/sys/ansi.h#L93-L114

There's no ANSI_GREY

Simple mistake but I think you need to be able to build and test your changes locally if you want to be able to contribute. I usually would compile a thing like this change and ensure the proc fires and does what I think it should, doesn't crash the mud and all the ansi looks pretty. Once, you get a local instance running it's not too much work to run a short test on each change.

@cizra's instructions here worked great: https://github.com/sneezymud/sneezymud-docker#to-develop-using-a-clean-database